src/deviation.jl

```diff
@@ -27,6 +36,11 @@ function countne(a::AbstractArray, b::AbstractArray)
 end
 
 # squared L2 distance
+"""
+    sqL2dist(a::AbstractArray{T}, b::AbstractArray{T})
+
+Computes the Squared L2 distance between `a` and `b`.
```

Comment:
"squared"

Could you give the definition of this distance too (i.e. `sum((a - b).^2)`), and mention that this function is more efficient.